clients/cobol-lsp-vscode-extension/src/services/snippetcompletion/cobolSnippets.json

```diff
@@ -96,55 +203,197 @@
+  },
+  "CONTROL": {
+    "prefix": "CONTROL",
+    "body": ["*CONTROL ${1|MAP,NOMAP|)"],
+    "description": "*CONTROL is a Compiler option - that allows the user to limit (or not limit) the MAP output of a program compile.  https://publibfp.dhe.ibm.com/epubs/pdf/igy6pg40.pdf "
+  },
   "CONVERTING": {
     "prefix": "CONVERTING",
-    "body": ["CONVERTING ${1:item}"],
-    "description": "CONVERTING item"
+    "body": [
+      "INSPECT DATA-4 CONVERTING",
+      "\"${1:abc}\" TO \"${2:123}\"",
+      "AFTER INITIAL \"|\" BEFORE INITIAL \"/\"",
+      "END-INSPECT",
+      "$0"
+    ],
+    "description": "CONVERTING is a parameter of the INSPECT keyword - used to replace character values. https://www.ibm.com/docs/en/cobol-zos/6.4?topic=inspect-examples-statement"
+  },
+  "COPY": {
+    "prefix": "COPY",
+    "body": ["COPY ${1:copybookName}"],
+    "description": "COPY is used to insert source code - stored in a PDS or SCM into COBOL lines. Note that the JCL or Source Management System (Endevor, etc.) must be available and specified correctly. https://publibfp.dhe.ibm.com/epubs/pdf/igy6pg40.pdf"
+  },
+  "COPY REPLACING": {
+    "prefix": "COPY",
+    "body": [
+      "COPY \"${1:SUBPROG.CPY}\" REPLCAING",
+      "\t ==${2:()}== BY ==${3:lnk-}==."
+    ],
+    "description": "COPY REPLACING"
   },
   "CORR": {
     "prefix": "CORR",
     "body": ["CORR ${1:id}"],
     "description": "CORR id"
   },
+  "COUNT": {
+    "prefix": "COUNT",
+    "body": [
+      "UNSTRING ${2:var-1} DELIMITED BY \"${3:*}\" INTO ${4:var-2}",
+      "COUNT IN ${1:var-3}",
+      "END-UNSTRING",
+      "$0"
+    ],
+    "description": "Part of the UNSTRING verb, COUNT places the number of characters in a numeric variable (VAR-3 in the above example). https://publibfp.dhe.ibm.com/epubs/pdf/igy6pg40.pdf"
+  },
   "COUNT IN": {
     "prefix": "COUNT IN",
     "body": ["COUNT IN ${1:id}"],
     "description": "COUNT IN id"
   },
-  "COPY": {
-    "prefix": "COPY",
+  "CRT": {
+    "prefix": "CRT",
+    "body": ["CRT STATUS IS ${1:statusVar}"],
+    "description": "Part of the SPECIAL-NAMES paragraph/CONFIGURATION SECTION in the ENVIRONMENT DIVISION. CRT STATUS returns data regarding the status of a screen-based COBOL ACCEPT statement when an ON EXCEPTION statement is included in the ACCEPT statement."
+  },
+  "CURRENCY": {
+    "prefix": "CURRENCY",
     "body": [
-      "COPY \"${1:SUBPROG.CPY}\"",
-      "    REPLACING ==${2:()}== BY ==${3:lnk-}==."
+      "CURRENCY SIGN IS \"${1:\\$USD}\" WITH PICTURE SYMBOL \"${2:\\$}\"."
     ],
-    "description": "COPY REPLACING"
+    "description": "CURRENCY SIGN clause is defined in the SPECIAL-NAMES paragraph in the CONFIGURATION SECTION/ENVIRONMENT DIVISION. In the given example, the PIC character $ indicates that the currency sign $USD is to be substitued. Note that you may declare multiple curency signs for multiple currencies. https://publibfp.dhe.ibm.com/epubs/pdf/igy6pg40.pdf"
```

Comment:
"The CURRENCY SIGN clause is defined in the SPECIAL-NAMES paragraph in the CONFIGURATION SECTION of the ENVIRONMENT DIVISION. In the given example, the PIC character $ indicates that the currency sign $USD is to be substituted. You can declare multiple currency signs for multiple currencies. https://publibfp.dhe.ibm.com/epubs/pdf/igy6pg40.pdf"